app/src/main/java/com/example/nvdovin/weatherapp/adapter/RecyclerViewAdapter.java

```diff
@@ -45,20 +45,20 @@ public CustomViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(CustomViewHolder holder, int position) {
-        City city = cityList.get(position);
-        String weatherIconId = city.getWeatherDataList().get(0).getWeatherIcon();
-        holder.cityName.setText(city.getName());
-        holder.cityWeatherDescription.setText(city.getWeatherDataList().get(0).getWeatherDescription());
-        holder.cityTemperature.setText(String.valueOf(city.getWeatherDataList().get(0).getTemp().intValue() - tempScale));
-        holder.cityID = city.getId();
+        ForecastNow forecastNow = forecastNowList.get(position);
+        String weatherIconId = forecastNow.getCurrentCityWeather().getWeatherIcon();
```

Comment:
getCurrentWeather x3 => extract to a local final variable so it will be easier to read the code.
rename the ForecasNow class in something more suggestive.